python/activator/activator.py

```diff
@@ -269,6 +408,13 @@ def parse_next_visit(http_request):
     if not event.data:
         raise ValueError("empty CloudEvent received")
 
+    if event['time']:
```

Comment:
Does this actually work if the field does not exist (I'm having trouble finding more than the most basic CloudEvent docs), or do you need to do something like:
```suggestion
    if 'time' in event:
```

Reply:
From @dspeck1 on Slack:
> Removed the Knative event time and can handle with a separate ticket/PR.  I had assumed the event time header was not sent, but it is.  It is in date time format so will probably just convert it without changing fan out.